app.go

```diff
@@ -8,11 +8,13 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/distributeddesigns/currency"
 	"github.com/op/go-logging"
 
-	"./accounts"
-	"./commands"
-	"./quotecache"
+
+	"Milestone1/accounts"
+	"Milestone1/commands"
+	"Milestone1/quotecache"
```

Comment:
Interesting. If I change it to `./whatever` it gives me a `local import of non-localfile exception`.

http://stackoverflow.com/questions/30885098/go-local-import-in-non-local-package